topics/statistics/tutorials/fine_tuning_protTrans/tutorial.md

```diff
@@ -0,0 +1,121 @@
+---
+layout: tutorial_hands_on
+
+title: Fine tune large protein model (ProtTrans) using HuggingFace
+zenodo_link: https://zenodo.org/records/10986248
+questions:
+- How to load large protein AI models?
+- How to fine-tune such models on downstream tasks such as post-translational site prediction?
+objectives:
+- Learn to load and use large protein models from HuggingFace
+- Learn to fine-tune them on specific tasks such as predicting dephosphorylation sites
+requirements:
+  -
+    type: internal
+    topic_name: galaxy-interface
+    tutorials:
+      - jupyterlab
+  -
+    type: internal
+    topic_name: statistics
+    tutorials:
+      - fine_tune_large_protein_models
+time_estimation: 1H
```

Comment:
this is generating an error, "Internal requirement to topics/statistics/tutorials/fine_tune_large_protein_models does not exist", is this link expected?

Reply:
yes, thanks. Just fixed it :)